index.html

```diff
@@ -0,0 +1,631 @@
+                </div>
+
+                <div class="griditem2">
+                    <img src="images/grid12.jpg" alt="image here">
+                    <a href="#">L. Calçada/ESO</a>
+                </div>
+                <div class="griditem2">
+                    <img src="images/grid13.jpg" alt="image here">
+                    <a href="#">Image by Joe Skipper/Reuters</a>
+                </div>
+                <div class="griditem2">
+                    <img src="images/grid14.jpg" alt="image here">
+                    <a href="#">NASA/JPL-Caltech/University of Arizona</a>
+                </div>
+                <div class="griditem2">
+                    <p>L. Calçada/ESO
+                        A Star Went Supernova in 1987. Where Is It Now?
+                        Aug. 8</p>
+                    <span class="date">Aug. 14</span>
+                </div>
+                <div class="griditem2">
+                    <p>Image by Joe Skipper/Reuters
+                        NASA Launches Perseverance Rover, Capping Summer of Missions to Mars
+                        July 30</p>
+                    <span class="date">Aug. 14</span>
+                </div>
+                <div class="griditem2">
+                    <p>NASA/JPL-Caltech/University of Arizona
+                        How NASA Found the Ideal Hole on Mars to Land In
+                        July 30</p>
+                    <span class="date">Aug. 14</span>
+                </div>
+            </div>
+
+            <hr>
+            <a href="#">Editor's Picks</a>
+            <div class="grid">
+                <div class="griditem2">
+                    <img src="images/grid9.jpg" alt="image here">
+                    <a href="#">European Southern Observatory</a>
+                </div>
+                <div class="griditem2">
+                    <img src="images/grid10.jpg" alt="image here">
+                    <a href="#">Bill Ingalls/NASA, via Agence France-Presse — Getty Images</a>
+                </div>
+                <div class="griditem2">
+                    <img src="images/grid11.jpg" alt="image here">
+                    <a href="#">Peter Komka/MTI, via Associated Press</a>
+                </div>
+                <div class="griditem2">
+                    <p>This Star Looked Like It Would Explode. Maybe It Just Sneezed Aug. 14</p>
+                    <span class="date">Aug. 14</span>
+                </div>
+                <div class="griditem2">
+                    <p>Amazon Satellites Add to Astronomers’ Worries About the Night Sky Aug. 10</p>
+                    <span class="date">Aug. 14</span>
+                </div>
+                <div class="griditem2">
+                    <p>‘Thanks for Flying SpaceX’: NASA Astronauts Safely Splash Down After Journey From Orbit Aug. 2
+                    </p>
+                    <span class="date">Aug. 14</span>
+                </div>
+            </div>
+
+
+        </div>
+
+
+
+
+
+
```

Comment:
Please avoid using redundant empty lines between tags and avoid using extra indentation (2 spaces suggested for each indentation). To provide this, I can suggest you to install the proper code-formatter plugins.